### PR TITLE
18th PR

### PR DESCRIPTION
**Pull request by makkoncept**

### CODE REVIEW

File: pulp-fiction [hunk 1]
```diff
@@ -1 +1,3 @@
 best of Tarantino?
```
Comment (ghost) [inline pulp-fiction:1]: comment

**Review by ghost — COMMENTED** [review]
comment - 1

File: pulp-fiction [hunk 2]
```diff
@@ -1 +1,3 @@
 best of Tarantino? 
+
+yes! with kill bill at close 2nd
```
Comment (makkoncept) [inline pulp-fiction:3]: comment

File: pulp-fiction (hunk 1 above)
Comment (makkoncept) [reply pulp-fiction:1]: comment

**Review by makkoncept — COMMENTED** [review]
comment

**Pull request closed without merging**